.gitignore

```
@@ -0,0 +1,2 @@
+.DS_Store
+.DS_Store?
```

1行でよいですね。あと末尾で改行しましょう
```suggestion
.DS_Store
```